Detectors/ITSMFT/ITS/reconstruction/src/CookedTracker.cxx

```diff
@@ -307,19 +292,19 @@ void CookedTracker::makeSeeds(std::vector<CookedTrack> &seeds, Int_t first, Int_
   Int_t nClusters3 = layer3.getNumberOfClusters();
 
   for (Int_t n1 = first; n1 < last; n1++) {
-    Cluster* c1 = layer1.getCluster(n1);
+    const Cluster* c1 = layer1.getCluster(n1);
     //
     // Int_t lab=c1->getLabel(0);
     //    
     Double_t z1 = c1->getZ();
-    auto xyz1 = c1->getXYZGloRot(*mGeom);
+    auto xyz1 = ((Cluster*)c1)->getXYZGloRot(*mGeom); // FIXME: Cannot this method be const ?
```

Comment:
@iouribelikov : Indeed, my fault. I pushed fix for constness in private branch:  https://github.com/shahor02/AliceO2/commit/d60a65e84ddd185cbf3968157094c9ef36b3d3c9, the fastest will be if you cherrypick this commit to your branch, rather than I open PR.

Reply:
@shahor02  OK. I'll do it.  Thanks a lot !